chia/util/initial-config.yaml

```diff
@@ -164,6 +164,7 @@ seeder:
 
 harvester:
   # The harvester server (if run) will run on this port
+  # TODO: this seems to be used, but i don't have the port open.  not sure what's going on
```

Comment:
I think this is just used because we create a `ChiaServer` for the harvester but we don't connect to it iirc.

Reply:
Thanks.  That's definitely backwards and should be a parameter or handled entirely outside.  The generic server should not be what is special casing the various usages of it.

Reply:
And all those port configs should be fully removed.